utilities/general-helpers.metta

```diff
@@ -9,7 +9,8 @@
 
 ;Function to cocatinate two tuples (A B) (C D) ==> (A B C D)
 ;; (: concatTuple (-> Expression Expression Expression))
- (= (concatTuple $x $y) (collapse (union (superpose $x) (superpose $y)))) ;; FIX: No longer working on version 0.2.3
+;  (= (concatTuple $x $y) (collapse (union (superpose $x) (superpose $y)))) ;; FIX: No longer working on version 0.2.3
+(= (concatTuple $x $y) (union-atom $x $y)) ;; This should work when run after activating the virtual env
```

Comment:
I don't think `union-atom` will be a permanent fix. Can you use a recursive definition for it instead?